packages/rainbowkit/package.json

```diff
@@ -57,10 +57,12 @@
     "vitest": "^0.5.0"
   },
   "dependencies": {
+    "@toruslabs/torus-embed": "^1.23.2",
     "@vanilla-extract/css": "1.7.0",
     "@vanilla-extract/dynamic": "2.0.2",
     "@vanilla-extract/sprinkles": "1.4.1",
     "clsx": "1.1.1",
+    "pnpm": "^7.1.7",
```

Comment:
We shouldn't need `pnpm` as a dependency here.

Reply:
my bad I send  the correction